### ISSUE TITLE
Class 'FOS\UserBundle\FOSUserBundle' not found in app/AppKernel.php line 20

### ISSUE THREAD

**Issue by diegotham** [FriendsOfSymfony/FOSUserBundle#2196]
I'm having this issue in production:
Class 'FOS\UserBundle\FOSUserBundle' not found in app/AppKernel.php line 20

It's the only bundle with the "dev-master" tag (~2.0@dev). If I put another bundle with a dev-master tag, it gives an error with that one, so it doesn't let me have any dev-master bundles.

I have tried many things: 
- Remove vendors, caches, bootstrap.php.cache, composer.lock
- Install composer with --no-dev, --dev... And with different combinations
- Add an env variable SYMFONY_ENV=prod before composer install
- Update composer, update composer dependencies

If I try to use the latest release: v2.0.0-alpha3, it gives an error with incompatibility with Symfony 3.0.

¿Is there a way to bypass this? Use a stable version of the bundle that works with Symfony 3

I'm using docker and beanstalk and I'm installing vendors before deploying.

**Comment by napestershine** [comment]
did you run install FOS with composer on production?

**Comment by diegotham** [comment]
The problem persisted doing that too.

I think I figured out the problem. It was weird because one of the points of Docker is to have exactly the same environment than in production, and it was. But I didn't realised that FOSUserBundle and 2 or 3 other dependencies where not being commited with Git when pushing/pulling. I used to do composer install/update in production so I never had this problem.

I tried with solutions like in here http://stackoverflow.com/questions/4185365/no-submodule-mapping-found-in-gitmodule-for-a-path-thats-not-a-submodule, but didn't work for me.

For now I'm just cloning the repo to the folder /vendor/friendsofsymfony/user-bundle, until I find a better way to deal with it like with submodules https://git-scm.com/book/en/v2/Git-Tools-Submodules.
